index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
♻️ Dear @ismayito. Your project is going well. However, along with the mentioned changes, there are some other things you might need to work on to make your project awesome. Remember that this project is the base for many projects in your Microverse Path. 

- [ ] Kindly install and configure the linters per [the guides](https://github.com/microverseinc/linters-config). Linters help you find bugs and errors even before comparing/shipping code. Those are great tools to help you write neat code ⭐. Please set the HTML and CSS linters. 
- [ ] Pease set up the GitHub actions as per the same guide. This will trigger the linters check so when you create a pull request you will be notified if there is a linter error. This check should always be green ✅  in order to continue with the  PR review.
- [ ] When working with JS and installing packages with npm (like the linters) you will notice a huge folder `node_modules` it is a best practice to add that folder to a `.gitignore` file 🧐. That way you avoid shipping code that is not needed in your repository. Please create a`.gitignore` file and add the `node_modules` so it is not tracked by git
- [ ] Another useful feature of the `.gitignore` file is that you can avoid pushing to the repo files that you don't actually want (testing files, sensitive config files, etc.). This is the case of the required `test.md` file. Kindly, please create that file locally, and add it to the `.gitignore` file so it is not shipped along with your code.

[OPTIONAL] Lastly, you did a good job with your commit messages describing what you did. However, remember that _The commit message should be written in the imperative tense and capitalized._. Please bear this in mind for your future projects! 🤠